styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -484,22 +489,26 @@ private String apiUrl(CharSequence... parts) {
   }
 
   private ByteString send(Request request) throws ExecutionException, InterruptedException {
-    return sendAsync(request).toCompletableFuture().get();
+    // TODO: Move requireSuccess use into callers of send()
```

Comment:
Yep, that was my point. I think that might not belong in `send`. Just asking your plan when to move it out.